examples/check-downloaded-file-name-and-content/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Verify file's contents</title>
```

Comment:
`Verify the downloaded file information`

Reply:
edited